app/models/miq_report/generator.rb

```diff
@@ -337,7 +337,6 @@ def generate_basic_results(options = {})
       :filter           => conditions,
       :include_for_find => get_include_for_find_rbac,
       :references       => get_include_rbac,
-      :where_clause     => where_clause,
```

Comment:
Slightly about this change and why this was needed before, and if it isn't going to break something by removing it.

I have my hands in a bit of this code here (the `:extra_cols` bit below), so I am somewhat familiar with it, but unsure how `:where_clause` ties into `Rbac` and how removing this now when it was being merged above previously as well is going to affect things.

Reply:
I was pretty sure that we could apply the where here, or we can pass it in for it to then be applied for us later

Reply:
I guess this way, this condition gets passed into calc_filtered_ids (if apply_rbac_directly / apply_rbac_through_association

the other way, we get those ids and then apply this condition after the fact.

Reply:
Okay, after looking over this again, I think I had it in my head that you were editing a different method here, but that is _**NOT**_ the case.  So I was just confused when asking this question originally...

So now, instead of defining `where_clause` and passing it as an `rbac_option`, we are just defining it on the `targets` scope, and that should have the same result.  I am cool with that change then.  Thanks!